core/src/main/java/jenkins/security/MasterToSlaveCallable.java

```diff
@@ -31,7 +31,7 @@ public void checkRoles(RoleChecker checker) throws SecurityException {
     public Channel getChannelOrFail() throws ChannelClosedException {
         final Channel ch = Channel.current();
         if (ch == null) {
-            throw new ChannelClosedException(new IllegalStateException("No channel associated with the thread"));
+            throw new ChannelClosedException(ch, new IllegalStateException("No channel associated with the thread"));
```

Comment:
Since ch is null, adding it to the Exception does not make any sense.

Reply:
This is to address the fact that the methods w/ fewer arguments are all deprecated. https://github.com/jenkinsci/remoting/blob/445c5a94c13ea42e6144e5a5db544c1cc23b0b5c/src/main/java/hudson/remoting/ChannelClosedException.java#L28

The alternative is me writing `(cast)null`, which seemed to be even less sensical.